.env.example

```diff
@@ -26,6 +26,10 @@ AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
 AWS_REGION=
 
+# redis details
+REDIS_HOSTNAME=
+REDIS_PORT=
```

Comment:
better to keep some default values that one needs to set to run this locally - just as with DB_PORT and DB_HOST